### ISSUE TITLE
Install Git-LFS on windows

### ISSUE THREAD

**Issue by valerocvm** [douglas-hoen/minion-red#3]
We need to find out how to install Git-LFS on Windows!

**Comment by douglas-hoen** [comment]
Yes. Instructions are [here](https://docs.github.com/en/github/managing-large-files/installing-git-large-file-storage) 

Unfortunately I have no way to try it, as I don't have a Windows system. If anyone wants to give it a try beforehand, that would save time at the meeting.
